src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -159,7 +159,7 @@ def test_resource_id_scenario(self, resource_group):
 
 class ResourceGenericUpdate(LiveScenarioTest):
     @ResourceGroupPreparer(name_prefix='cli_test_resource_id')
-    def test_resource_id_scenario(self, resource_group):
+    def test_resource_id_scenario2(self, resource_group):
```

Comment:
you can rename it back and also delete the recording file since it is a live test

Reply:
I changed the name because there is another `test_resource_id_scenario` test and azdev logs an error (and as a result won't run either).

Reply:
I changed the name to match the class (test_resource_generic_update). Since it uses --ids to run things in parallel, I have to keep it Live only.